content/internships/applicant-guide/contents.lr

```diff
@@ -49,6 +50,11 @@ Please read our [Contributing Code](/contributing-code) page for information on
 
 If you're applying via Outreachy, remember to record your contribution via the Outreachy website! You can record contributions even if you have not yet received feedback on them or if they have not been merged – our mentors are busy so we can be slow to provide feedback.
 
+#### Contributing Documentation
```

Comment:
shouldn't this just be three hashes?

Reply:
I meant for it to be under "How to Contribute" since it's mostly about _how_ we would like to see documentation contributions made. I can make it three hashes though, if that makes more sense to you :)